app/services/stack_overflow.rb

```diff
@@ -9,30 +9,37 @@ def initialize(term="ruby")
 
   def scrape
     feed = pull_feed
-    if feed.entries.present?
-      format_entries(feed.entries)
+    if feed.present?
+      format_feed(feed)
     end
   end
 
-  def format_entries(entries)
-    entries.map do |entry|
-      formatted_entry = self.format_entry(entry)
+  def format_feed(feed)
+    feed.css('item').map do |item|
+      formatted_entry = self.format_entry(item)
       create_records(formatted_entry)
     end
   end
 
   def format_entry(entry)
+    title = entry.css('title').text
+    url_address = entry.css('guid').text
+    location = entry.css('location').text
+    description = entry.css('description').text
     { job: {
-        title: entry.title,
-        url: entry.url,
-        location: self.pull_location(entry.title),
+        title: title,
+        url: url_address,
+        location: location,
         raw_technologies: generate_raw_technologies(entry), #["perl", "python", "ruby", "or-go.-ruby-and", "or-go-experience-is-stron"],
-        description: entry.summary,
-        remote: self.is_remote?(entry.title),
-        posted_date: entry.published
+        description: description,
+        remote: self.is_remote?(title),
+        posted_date: entry.css('pubdate').text
       },
       company: {
-        name: self.pull_company_name(entry.title)
+        name: self.pull_company_name(title)
+      },
+      location: {
```

Comment:
There doesn't seem to be a need for this currently. You load the location as part of jobs (as in line 32).